samples/client/petstore/java/apache-httpclient/src/main/java/org/openapitools/client/model/NullableClass.java

```diff
@@ -81,7 +81,7 @@ public class NullableClass extends HashMap<String, Object> {
   private JsonNullable<List<Object>> arrayAndItemsNullableProp = JsonNullable.<List<Object>>undefined();
 
   public static final String JSON_PROPERTY_ARRAY_ITEMS_NULLABLE = "array_items_nullable";
-  private List<Object> arrayItemsNullable = null;
+  private List<Object> arrayItemsNullable = new ArrayList<>();
```

Comment:
❓ If the field is `nullable` shouldn't it be `null` here?

Reply:
the array item is nullable, not the array itself:
```
        array_items_nullable:
          type: array
          items:
            type: object
            nullable: true
```

Reply:
got it